src/components/hack-details.js

```diff
@@ -102,14 +102,9 @@ export function HackDetails({ match, dispatch, userId, hackDetails }) {
               <Grid item xs={10}>
                 <Typography variant="body1">{hackDetails.goal}</Typography>
               </Grid>
-              {hackDetails.team !== undefined ? (
-                <Grid item xs={10} className={classes.marginTop}>
-                  <Typography variant="h6">Team Members:</Typography>
-                </Grid>
+              {hackDetails.team.length !== 0 ? (
+                <Team team={hackDetails.team} name={hackDetails.teamName}/>
               ) : null}
```

Comment:
I'd let the `<Team>` component handle the empty state so we can do something like:
```
<p>Looks like no one is in this team yet :(<br />Want to be the first? <a onClick={joinTeam}>Join the team!</a></p>
```